src/Mvc/test/Mvc.FunctionalTests/ErrorPageTests.cs

```diff
@@ -183,9 +183,4 @@ public async Task AggregateException_FlattensInnerExceptions()
         Assert.Contains(nullReferenceException, content);
         Assert.Contains(indexOutOfRangeException, content);
     }
-
-    public void Dispose()
-    {
-        _assemblyTestLog.Dispose();
```

Comment:
This is important because individual tests don't own `AssemblyTestLog` instances, `AspNetTestAssemblyRunner` does